src/components/Header/HomePageHeader.tsx

```diff
@@ -20,7 +20,10 @@ const HomePageHeader = (props: { indicatorsLinkOnClick: () => void }) => {
   return (
     <Wrapper>
       <Content>
-        <HeaderTitle>America’s COVID Warning System</HeaderTitle>
+        <HeaderTitle component="h1">
+          US COVID Map &amp;{' '}
+          <span style={{ whiteSpace: 'nowrap' }}>Risk Levels</span>
```

Comment:
Is there a better way to do this?  Should I create a tiny helper `<NoLineBreak>` component or something?

Reply:
you can if you want to, but this looks fine to me !